### PR TITLE
OHIF-198 - Jump to thumbnail when a new displaySet is added

### PR DESCRIPTION
**Pull request by JamesAPetts**
WIP Jump to thumbnail when SR saved.

on displaySet Creation:
- jumps to the right tab
- opens the study in the browser if not open.
- scrolls to the thumbnail element.

### CODE REVIEW

File: platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
```
Comment (dannyrb) [inline platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx:3]: You ended up not using this

File: extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx [hunk 2]
```diff
@@ -34,6 +34,7 @@ function PanelStudyBrowserTracking({
   const [studyDisplayList, setStudyDisplayList] = useState([]);
   const [displaySets, setDisplaySets] = useState([]);
   const [thumbnailImageSrcMap, setThumbnailImageSrcMap] = useState({});
+  const [jumpToDisplaySet, setJumpToDisplaySet] = useState(null);
```
Comment (dannyrb) [inline extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx:37]: I don't think we need this

File: extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx [hunk 3]
```diff
@@ -138,16 +139,23 @@ function PanelStudyBrowserTracking({
...
         newDisplaySets.forEach(async dSet => {
+          const displaySetInstanceUID = dSet.displaySetInstanceUID;
+
           const newImageSrcEntry = {};
           const displaySet = DisplaySetService.getDisplaySetByUID(
-            dSet.displaySetInstanceUID
+            displaySetInstanceUID
           );
+
+          if (dSet.madeInClient) {
```
Comment (dannyrb) [inline extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx:149]: I'm not sure I want to expand `displaySet`'s props to support this specifically. If there are other benefits, we can talk about keeping it?

File: extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx [hunk 4]
```diff
@@ -401,3 +453,24 @@ function _createStudyBrowserTabs(
 
   return tabs;
 }
+
+function _findTabAndStudyOfDisplaySet(displaySetInstanceUID, tabs) {
```
Comment (dannyrb) [inline extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx:458]: You have `StudyInstanceUID` at time of save. Depending on how we order thumbnails, the newly created SR will either be the first or last SR in the list of thumbnails.

I think we can key most of this off either the current subscription, or identify a new platform level event that makes sense for what's occurring, and then accomplish all of this. Unfortunately, for scrollIntoView, each thumbnail may need a `ref`

File: extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx (hunk 3 above)
Comment (JamesAPetts) [reply extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx]: Changed, now on event.

File: extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx (hunk 4 above)
Comment (JamesAPetts) [reply extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.jsx:459]: Kept generic for now as discussed.

**Pull request merged**